_includes/code/howto/java/src/test/java/io/weaviate/docs/manage-data.classes.java

```diff
@@ -72,6 +72,34 @@ private void createCollection(String className) {
       .returns(true, Result::getResult);
   }
 
+  private void createCollectionWithProperties(String className){
+    // START CreateCollectionWithProperties"
```

Comment:
It's probably better to show the whole class. Consider moving the // START line before the private declaration